src/ResourceManager/Advisor/Commands.Advisor/help/Set-AzureRmAdvisorConfiguration.md

```diff
@@ -0,0 +1,218 @@
+---
+external help file: Microsoft.Azure.Commands.Advisor.dll-Help.xml
+Module Name: Az.Advisor
+online version:
+schema: 2.0.0
+---
+
+# Set-AzureRmAdvisorConfiguration
+
+## SYNOPSIS
+Updates or creates the Aure Advisor Configration.
+
+## SYNTAX
+
+### InputObjectLowCpuExcludeParameterSet (Default)
+```
+Set-AzureRmAdvisorConfiguration [-Exclude] [-LowCpuThreshold] <String>
+ [[-InputObject] <PsAzureAdvisorConfigurationData>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### InputObjectRgExcludeParameterSet
+```
+Set-AzureRmAdvisorConfiguration [-Exclude] [[-ResourceGroupName] <String>]
+ [[-InputObject] <PsAzureAdvisorConfigurationData>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Used to update the configuration of the advisors. Two types of configration are present Subscription level configuration and ResourceGroup level configuration. 
+
+Subscription level configuration: There can be only one configration for this type, lowCpuThreshold and exclude properties can be updated using cmdlet.
+ResourceGroup level configuration: There can be only one configration for each ResourceGroup, only exclude property could be updated using cmdlet.
```

Comment:
ResourceGroup level configuration: There can be only one configuration for each ResourceGroup, and only the exclude property can be update using this cmdlet.